css/tech.scss

```diff
@@ -224,8 +212,6 @@
   .tech {
     .section-1 {
```

Comment:
Sorry for catching this now, but can you change this to double dashes so `section--1` here and elsewhere where there's a number after a block name